source/administration/monitoring.txt

```diff
@@ -354,6 +354,55 @@ affect logging:
 - :dbcommand:`logRotate`. Rotates the log files for :program:`mongod`
   processes only. See :doc:`/tutorial/rotate-log-files`.
 
+.. _monitoring-log-redaction:
+
+Log Redaction
+~~~~~~~~~~~~~
+
+.. versionadded:: 3.4 Available in MongoDB Enterprise only
+
+A :program:`mongod` running with :setting:`security.redactClientLogData`
+removes :doc:`messages </reference/log-messages>` associated with any given log
```

Comment:
Q: does this remove messages?  as in it first logs and then redacts messages? or is it that the redacted message is logged.